docs/api-docs/headless/end-to-end-guide/graphql.mdx

```diff
@@ -214,7 +214,7 @@ To add a new line item to the existing cart, add the cart ID to the GraphQL vari
     mutation addCartLineItems($addCartLineItemsInput: AddCartLineItemsInput!) {
       cart {
         addCartLineItems(input: $addCartLineItemsInput) {
-          cart {
+          cart(entityId: $entityId) {
```

Comment:
Same here:  It's not appropriate to add an argument referencing a variable that is not actually being passed to the mutation.

For this particular mutation - addCartLineItems - the ID of the cart being edited is already part of the $addCartLineItemsInput being passed in.  (See your example of variable values below.)  So this mutation should also remain as it was.